2019_03_22_InputOutputHome/src/Main.java

```diff
@@ -0,0 +1,147 @@
+import org.apache.commons.io.FileUtils;
+
+import java.io.*;
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class Main {
+	public static void main(String[] args) throws IOException {
+		List<Customer> customers = Arrays.asList(new Customer("Ivan","Ivanich", 22),
+				new Customer("Sergio", "Pedro", 25),
+				new Customer("Ishtvan", "Poladi", 79));
+
+
+
+		System.out.println("******************Just reading file***************************************");
+		FileReaderFromFile("/home/sabadosh/Documents/test.txt");
+
+		System.out.println("******************Summing all integers in file****************************");
+		sumIntFromFile("testWithIntegers");
```

Comment:
It seems like you still only print it, not really return the result and then print it.